scvi/model/_multivi.py

```diff
@@ -175,6 +175,12 @@ def __init__(
             REGISTRY_KEYS.SIZE_FACTOR_KEY in self.adata_manager.data_registry
         )
 
+        if n_proteins is None:
+            if n_proteins in self.summary_stats:
+                n_proteins = self.summary_stats.n_proteins
+            else:
+                n_proteins = 0
```

Comment:
```suggestion
        if n_proteins in self.summary_stats:
            n_proteins = self.summary_stats.n_proteins
        else:
            n_proteins = 0
```

Reply:
In the note of the docstring can you mention that n_proteins is inferred automatically?

Reply:
Also now you will need to fix the tests